outputs.tf

```
@@ -1,14 +1,14 @@
-output "s3logging_bucket_arn" {
+output "s3_bucket_arn" {
```

Is `s3` not thoroughly implied with just `bucket_arn`? I don't feel strongly about this but it seems this could be reduced a bit more.

```suggestion
output "bucket_arn" {
```

---

We have an outputs naming policy that says include the resource name minus the provider. That said, I'm starting to feel like it could be better because it creates an awkward situation where a outputs derived from a module's outputs become very compact and non-type centric, while a resource or module's direct outputs are verbose and well typed.

I don't think this is the project to fix that, but I definitely agree with you this is verbose without adding value. The goal was to just be able to have consistent output names.